symengine/cwrapper.cpp

```diff
@@ -658,6 +662,164 @@ void basic_str_free(char *s)
     delete[] s;
 }
 
+void bool_true(basic s)
```

Comment:
`basic_set_bool_true`?

Reply:
Or `basic_bool_set_true`?

Reply:
Ok, we have `integer_set_si` so I changed to `bool_set_true` and `bool_set_false`